.github/workflows/pypi_publish.yml

```diff
@@ -48,8 +59,122 @@ jobs:
           name: wheels-sdist
           path: ./wheelhouse/*.tar.gz
 
+  build-wheels:
+    name: 'Wheels: ${{ matrix.os }} ${{ matrix.arch }} ${{ matrix.python-version }}'
+    runs-on: ${{ matrix.os }}
+    continue-on-error: true
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13']
```

Comment:
Looking at the h5py workflow, they don't use separate python versions in it. Instead, they use a simple "*" (the default in `tiage_build.sh` and then `pypa/cibuildwheel` deals with the multiple iterations.

Reply:
Yep, I've seen this done either way, so whatever is preferable is fine by me.